CMakeLists.txt

```diff
@@ -45,6 +45,20 @@ option(FINUFFT_BUILD_DEVEL "Whether to build developement executables" OFF)
 option(FINUFFT_ENABLE_INSTALL "Whether to enable installation of FINUFFT library" ON)
 # sphinx tag (don't remove): @cmake_opts_end
 
+set(FFTW_ROOT $ENV{FFTWDIR})
+
+message("FFTW_ROOT=${FFTW_ROOT}")
+
+find_library(
+    FFTW_DOUBLE_LIB
+    NAMES "fftw3" libfftw3-3
+    PATHS ${FFTW_ROOT}
+    PATH_SUFFIXES "lib" "lib64"
+    NO_DEFAULT_PATH
+    )
+
+message("FFTW_DOUBLE_LIB=${FFTW_DOUBLE_LIB}")
```

Comment:
@janden 

I do not think this is needed, [this](https://github.com/flatironinstitute/finufft/blob/4b5d452df027feeb4cda6ba10fd922a55423f2ef/cmake/setupFFTW.cmake#L1) handles it. You just need to set `FFTW_ROOT` [source](https://github.com/egpbos/findFFTW?search=1)

Reply:
Yes this was just to debug. Mistakenly pushed to the main branch.